mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -977,11 +997,16 @@ resource "google_container_cluster" "with_gcp_public_cidrs_access_enabled" {
     gcp_public_cidrs_access_enabled = %s
   }
   deletion_protection = false
+  network    = data.google_compute_network.container_network.name
+}
+
+data "google_compute_network" "container_network" {
```

Comment:
IMO we can skip the datasource and supply the network name directly- we're just pulling the same value we piped in off of it.

Reply:
yeah, at first I thought we needed the id and that's why I created the datasource and then I just realized that I was actually using the name 🤦. I'll just remove them